### ISSUE TITLE
How to segment a long conversation?

### ISSUE THREAD

**Issue by jinserk** [kaldi-asr/kaldi#1746]
Hi,
I'm trying to decode the talkbank's callhome corpus, which contains 30-minute telephone conversations. The result is awesome using aspire chain pre-trained model and online2 decoder, using --online=true and --do-endpointing=false. Thanks for the great model!
But the output is mixture of sentences from two speakers, so I want to separate the result per speaker and per sentence, and to list them in time order. Since the wav file has separate channel per speaker, so I can split each channel to decode them twice per speaker. But I have no idea how to divide the result by sentence. 
In a quick thought, it counts long silence during the decoding, so I could insert any specific delimiter in the silence. 
Could you give me any clue to do so? Thank you!

**Comment by jinserk** [comment]
Thank you so much for your kind answers, @danpovey! As you said, I'm trying to learn from the resources and Kaldi codes as much as I can. 
I found that the ilabels output GetLinearSymbolSequence() contains lots of silence phones, so I can try to make such sentence segmentation. I also found there is segmentation.pl in utils directory, so looking into the script. However, once I cannot still understand how to make this input side segmentation related to the output word sequence.. Do I need to decode the segmented inputs once again?
